iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ReactiveCapabilityCurveImpl.java

```diff
@@ -88,9 +126,7 @@ public ReactiveLimitsKind getKind() {
 
     @Override
     public double getMinQ(double p) {
-        if (points.size() < 2) {
-            throw new IllegalStateException("points size should be >= 2");
-        }
+        checkPointsSize(points);
         Point pt = points.get(p);
```

Comment:
check should be done at build time not at each call. And again this is a potential regression.